packages/lib-react-components/src/CloseButton/CloseButton.stories.js

```diff
@@ -17,7 +17,7 @@ const config = {
 
 const darkZooTheme = { ...zooTheme, dark: true }
 
-storiesOf('CloseButton', module)
+storiesOf('Components/Modal/CloseButton', module)
```

Comment:
I don't think this should be nested under the `Modal` story. The `CloseButton` component is available and exported independently of it and therefore should have its own story. This here implies that the `CloseButton` is not and is simply a child component of the `Modal`

Reply:
No problem. `Modal` is the only component that uses the close button, so I assumed it would be nested under `Modal`. Note that `AdminCheckbox` is exported independently but has always been listed as part of `ZooFooter` in our component hierarchy.

Reply:
The way I'd correct that is to have the `AdminCheckbox` have its own story. `CloseButton` is used on its own in [our code](https://github.com/zooniverse/front-end-monorepo/blob/e0c725484953612b2c1f6808d8a5095a2fe0909c/packages/app-project/src/components/Announcements/components/GenericAnnouncement/components/WideGenericAnnouncement/WideGenericAnnouncement.js#L2).